### PR TITLE
docs: :memo: Move user-test.qmd from Sprout to Teams repo

### PR DESCRIPTION
**Pull request by K-Beicher**
## Description

- This PR finds a better home for the user-test.qmd document, moving it from Sprout to Teams.
- These changes are done because we want to use the user test document in more than one dev repo, and the teams repo seems to be the most natural place to place it (at least for now).

## Related Issues

Closes Sprout seedcase-project/seedcase-sprout#424

## Reviewer Focus

This PR only needs a quick review.

First time moving a document, please note that the original has not yet been deleted, that will be done once I know the document is sitting correctly in Teams.

## Checklist

For general documentation:

- [ ] Spell-check
    - [ ] US
    - [ ] UK
- [ ] Did the page(s) preview correctly on your machine without breaking
- [ ] New category words (keywords) (if any) added to the code snippet file

### CODE REVIEW

**Review by signekb — COMMENTED** [review]
Nice 👌 
Should this file be in the `entries` folder instead of creating a new `docs/uer-tests` directory?

**Comment by K-Beicher** [comment]
Good point @signekb, @lwjohnst86 where would you put it?

**Comment by lwjohnst86** [comment]
Hmmm, excellent question... I think maybe `user-tests/sprout.qmd`? I'll make some edits anyway, so I can move it over.

File: user-tests/sprout.qmd [hunk 1]
```diff
@@ -1,103 +1,108 @@
...
+    and `physical_activity_2024.csv`.
 
-| id  | physical_activity | duration | date  |
-|-----|---------------|----------|-----------|
-| 1  | run           | 10       | 01-03-2024 |
-| 2  | walk          | 25       | 02-04-2024 |
+| id  | physical_activity | duration | date       |
+|-----|-------------------|----------|------------|
+| 1   | run               | 10       | 01-03-2024 |
+| 2   | walk              | 25       | 02-04-2024 |
```
Comment (signekb) [inline user-tests/sprout.qmd:66]: @lwjohnst86 Did you do this manually?

**Pull request merged**